enterprise-suite/values.yaml

```diff
@@ -79,7 +71,7 @@ imageCredentials:
 createAlertManager: true
 
 # Comma separated list of alertmanager addresses (can be k8s local DNS service names)
-alertManagers: alertmanager:9093
+alertManagers: alertmanager
```

Comment:
Removing port number here needs to also be reflected in the docs: https://github.com/lightbend/es-docs/blob/master/src/main/paradox/installation/alertmanager.md#installing-with-existing-alertmanager

Reply:
Fixed it to not use port.